.github/workflows/galaxy-importer.cfg

```diff
@@ -6,4 +6,5 @@ RUN_ANSIBLE_LINT = True
 RUN_ANSIBLE_TEST = False
 ANSIBLE_TEST_LOCAL_IMAGE = False
 LOCAL_IMAGE_DOCKER = False
-INFRA_OSD = False
\ No newline at end of file
+INFRA_OSD = False
+CHECK_CHANGELOG = False
```

Comment:
Is there a reason we do not do this `CHECK_CHANGELOG` ansible aci?

Reply:
The changelog file had incorrect indenting in MSO but in ND and ACI the indenting was fine. 🤔 
I thought this setting would ignore it but it did not. I will revert this line.

Reply:
Done